README.md

```diff
@@ -37,10 +37,16 @@ Contains crates specific to our application. Can depend on libraries located in
 ## Build & Run
 
 1. `cargo build` (do `export SOLC_BIN=/usr/bin/solc` if `solc` is installed locally)
```

Comment:
the second part should be outdated after #951 is merged

Reply:
Did this suggestion in https://github.com/comit-network/comit-rs/pull/951 as it is related to that task. 
Details: https://github.com/comit-network/comit-rs/pull/951/commits/10c312bdddd1a3f6e3366b3b4ee076db47fd2fbc